src/engraving/layout/layoutbeams.h

```diff
@@ -30,16 +30,16 @@ class ChordRest;
 }
 
 namespace mu::engraving {
-struct LayoutContext;
+class LayoutStateContext;
 class LayoutBeams
 {
 public:
 
     static bool isTopBeam(Ms::ChordRest* cr);
     static bool notTopBeam(Ms::ChordRest* cr);
-    static void createBeams(Ms::Score* score, struct LayoutContext& lc, Ms::Measure* measure);
+    static void createBeams(Ms::Score* score, struct LayoutStateContext& lc, Ms::Measure* measure);
```

Comment:
did you mean const LayoutStateContext& lc?

Reply:
I will fix it in a next PR